imageio/plugins/ffmpeg.py

```diff
@@ -662,20 +662,39 @@ def run(self):
 
 def parse_device_names(ffmpeg_output):
     """ Parse the output of the ffmpeg -list-devices command"""
+    # Collect device names - get [friendly_name, alt_name] of each
     device_names = []
     in_video_devices = False
     for line in ffmpeg_output.splitlines():
         if line.startswith("[dshow"):
             logger.debug(line)
             line = line.split("]", 1)[1].strip()
             if in_video_devices and line.startswith('"'):
-                device_names.append(line[1:-1])
+                friendly_name = line[1:-1]
+                device_names.append([friendly_name, ""])
+            elif in_video_devices and line.lower().startswith("alternative name"):
+                alt_name = line.split(" name ", 1)[1].strip()[1:-1]
+                if sys.platform.startswith("win"):
+                    alt_name = alt_name.replace("&", "^&")  # Tested to work
+                else:
+                    alt_name = alt_name.replace("&", "\\&")  # Does this work?
```

Comment:
@One-sixth I was able to test this on Windows. Do you happen to be on Unix? If so, it would be great if you could give this a try.